las/client.py

```diff
@@ -1166,7 +1168,12 @@ def update_training(
         :raises: :py:class:`~las.InvalidCredentialsException`, :py:class:`~las.TooManyRequestsException`,\
  :py:class:`~las.LimitExceededException`, :py:class:`requests.exception.RequestException`
         """
-        return self._make_request(requests.patch, f'/models/{model_id}/trainings/{training_id}', body=optional_args)
+        body = {**optional_args}
+        if 'deployment_environment_id' in optional_args:
+            body['deploymentEnvironmentId'] = optional_args['deployment_environment_id']
+            del body['deployment_environment_id']
+
```

Comment:
IMO it feels weird to first add and then correct. I would suggest the following pattern:

```
if dep_id := optional_args.pop('deployment_environment_id', None):
    optional_args['deploymentEnvironmentId'] = dep_id 
body = {**optional_args}
```

Reply:
There are a few problems with this approach.

1) Walrus operator is not supported in Python 3.6
2) It is valid to set deployment environment to `null` and your code example will not allow this possibility

But I agree that we can use `.pop` instead here, see updated PR